src/Sylius/Bundle/ResourceBundle/test/src/Tests/Controller/ComicBookApiTest.php

```diff
@@ -95,31 +116,55 @@ public function it_allows_showing_a_comic_book()
     {
         $objects = $this->loadFixturesFromFile('comic_books.yml');
 
-        $this->client->request('GET', '/comic-books/'. $objects["comic-book1"]->getId());
+        $this->client->request('GET', 'v1/comic-books/'. $objects["comic-book1"]->getId());
```

Comment:
After adding version to the paths, there are absolute paths `/v1/comic-books/` and relative paths `v1/comic-books/` all over the API tests, we should unify that (probably to absolute ones).